openlibrary/catalog/marc/parse.py

```diff
@@ -668,6 +687,12 @@ def read_edition(rec):
         update_edition(rec, edition, read_languages, 'languages')
         update_edition(rec, edition, read_pub_date, 'publish_date')
 
+    saved_language = edition['languages'][0] if 'languages' in edition else None
+    update_edition(rec, edition, read_languages, 'languages')
+    if 'languages' in edition and saved_language not in edition['languages']:
+        # This shouldn't happen, but just in case
```

Comment:
I don't know how to handle this either. I imagine it does occur, but I'd need to see some examples to figure out what the best way to proceed. I imagine the `008` could have a generic or uncoded value (is `mul` a code for multiple?), and `041` could be more specific in some cases. 

I'm convincing myself that this logic should eventually be handled by `read_languages()`, but that we'd need to review and possibly look for more real test data to confirm, after #7410, and other language changes.

Reply:
This can definitely happen through human error. The rules are that the first language should be the language from the `008` (if it's coded), but it's just a rule for a human cataloger to follow. I'll change the comment to make it clearer.

`mul` in `008` means no predominant language whereas `mul` in `041` is used when there's more than some large number (6-7? )of languages. It might be that if there's `mul` in `008` and there are, say, 3 language codes in the `041` that they should replace, instead of being added to, the `mul` but I'm not familiar enough with cataloging practices to know.

https://www.oclc.org/bibformats/en/fixedfield/lang.html

Reply:
I've updated the comment to clarify that it happens in the case of human error.